addons/mail/models/discuss/discuss_channel.py

```diff
@@ -69,6 +69,8 @@ def _generate_random_token(self):
         'discuss.channel.member', 'channel_id', string='Members',
         groups='base.group_user')
     pinned_message_ids = fields.One2many('mail.message', 'res_id', domain=[('model', '=', 'discuss.channel'), ('pinned_at', '!=', False)], string='Pinned Messages')
+    sfu_channel_uuid = fields.Char(readonly=True, groups="base.group_system")
+    sfu_server_url = fields.Char(readonly=True, groups="base.group_system")
```

Comment:
probably better for debugging to not enforce readonly, and even to add the field on the form view in debug mode

Reply:
@seb-odoo I removed the readonly and added the fields in the view in the main PR (so that the fields are not in the views without the feature): https://github.com/odoo/odoo/compare/bbb311ec0c61f5d33c52bfa6a8eae72f4dd9bf1d..11aa7566da226ed8a77004864414f9be3f862cf1